### PR TITLE
Added a quick and simple guide for building integrations for Astro

### PR DESCRIPTION
**Pull request by leen-neel**
#### What kind of changes does this PR include?
- New or updated content

#### Description

Added a quick and simple guide that walks the reader through the process of creating a simple integration for Astro.

### CODE REVIEW

**Comment by sarah11918** [comment]
I was wondering why this didn't look so "quick and simple", @leen-neel :sweat_smile: 

Can you please remove the Ghost CMS guide from this PR? (If you're not sure how to do that, since we are both super-expert git users... I think I can delete the page from this PR myself. But, if you can do it, that's best.)  Thanks!  :smile:

**Comment by leen-neel** [comment]
@sarah11918 Yeah my git skills are really showing off right now 😅

I'll see into it ASAP

**Comment by leen-neel** [comment]
@sarah11918 There you go, fixed it!

**Comment by sarah11918** [comment]
Hi @leen-neel! I just wanted to let you know that we will have to put this one on hold a little longer since we'll be getting guides ready, and working on the docs site iteself for our upcoming v2 launch! 

We just have to prioritize getting some other material published and some other pages all sparklingly polished.I think we're going to have our hands full for the next little while and won't be able to give this one the polish it needs in time, so thanks for hanging in there, and we'll be back to it in the future!

**Comment by leen-neel** [comment]
@sarah11918 No problem! Take all the time you need

**Comment by wassfila** [comment]
@leen-neel given that the main has drifted quite a bit, your fork is 267 commits behind, could you rebase your fork so that we can read the PR preview with only you changes as delta ?

**Comment by leen-neel** [comment]
@wassfila On it!

**Comment by leen-neel** [comment]
@wassfila Done :D

File: src/pages/en/guides/cms/ghost.mdx [hunk 1]
```diff
@@ -285,5 +285,3 @@ To deploy your site visit our [deployment guide](/en/guides/deploy/) and follow
 
 - You can get started with this [Astro Ghost Starter](https://github.com/PhilDL/astro-starter-ghost).
 - Andre Carrera wrote [a blog post](https://andr.ec/posts/astro-ghost-blog/) showing how to use Ghost's SDK to embed posts and previews in your Astro project.
```
Comment (sarah11918) [inline src/pages/en/guides/cms/ghost.mdx:287]:
```suggestion
- Andre Carrera wrote [a blog post](https://andr.ec/posts/astro-ghost-blog/) showing how to use Ghost's SDK to embed posts and previews in your Astro project.


```

**Comment by sarah11918** [comment]
hey @leen-neel ! I'm so sorry that we couldn't prioritize this one. And in the mean time, there are too many conflicts in this PR for me to easily resolve, given how much has changed. We talked about this one, and we're going to close it for now.

We think a smaller, specific example would make a great recipe, though! What if instead of a generic guide to building an integration, you consider building one small integration as a recipe? Then, it should be easier to test since it will be specific instructions building an actual thing? General guides can be tough to get right, and we don't currently have the bandwidth to help you craft one.

If you're interested in trying a recipe instand, check out our new Recipes contributor guide!

**Pull request closed without merging**